frontend/src/styles/index.css

```
@@ -1,11 +1,39 @@
+
+@keyframes reveal {
+  0% { opacity: 0; transform: translateY(10px); filter: blur(5px); }
+  100% { opacity: 1; transform: translateY(0); filter: blur(0); }
+}
+
+@keyframes breathe {
+  0% { opacity: 0.4; transform: scale(0.95); }
+  100% { opacity: 1; transform: scale(1.05); }
+}
+
 @keyframes nameScroll {
```

Stylelint (17.13.0)

[error] Expected keyframe name "nameScroll" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)